CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 0.1.1 (TBD)
```

Comment:
Please add today's date as we're ready to launch this version after your PR.

Reply:
done!